tools/astarte_import/test/astarte/populatedb_test.exs

```diff
@@ -19,63 +19,94 @@
+  <credentials inhibit_request="false" cert_serial="324725654494785828109237459525026742139358888604" cert_aki="a8eaf08a797f0b10bb9e7b5dca027ec2571c5ea6" first_credentials_request="2019-05-30T13:49:57.355Z" last_credentials_request_ip="198.51.100.1"/>
+  <stats total_received_msgs="64" total_received_bytes="3960" last_connection="2019-05-30T13:49:57.561Z" last_disconnection="2019-05-30T13:51:00.038Z" last_seen_ip="198.51.100.89"/>
+  <interfaces>
+  <interface name="properties.org" major_version="0" minor_version="1" active="true">
+  <property reception_timestamp="2020-01-30T03:26:23.184Z" path="/properties1">42.0</property>
+  <property reception_timestamp="2020-01-30T03:26:23.185Z" path="/properties2">This is property string</property>
+  </interface>
+  <interface name="org.individualdatastreams.values" major_version="0" minor_version="1" active="true">
+  <datastream path="/testinstall1">
+  <value reception_timestamp="2019-05-31T09:12:42.789Z">0.1</value>
+  <value reception_timestamp="2019-05-31T09:13:29.144Z">0.2</value>
+  <value reception_timestamp="2019-05-31T09:13:52.040Z">0.3</value>
+  </datastream>
+  <datastream path="/testinstall2">
+  <value reception_timestamp="2019-05-31T09:12:42.789Z">3</value>
+  <value reception_timestamp="2019-05-31T09:13:52.040Z">4</value>
+  </datastream>
+  <datastream path="/testinstall3">
+  <value reception_timestamp="2019-05-31T09:12:42.789Z">true</value>
+  <value reception_timestamp="2019-05-31T09:13:29.144Z">false</value>
+  <value reception_timestamp="2019-05-31T09:13:52.040Z">true</value>
+  </datastream>
+  <datastream path="/testinstall4">
+  <value reception_timestamp="2019-05-31T09:12:42.789Z">This is the data1</value>
+  <value reception_timestamp="2019-05-31T09:13:29.144Z">This is the data2</value>
+  <value reception_timestamp="2019-05-31T09:13:52.040Z">This is the data3</value>
+  </datastream>
+  <datastream path="/testinstall5">
+  <value reception_timestamp="2019-05-31T09:12:42.789Z">3244325554</value>
+  <value reception_timestamp="2019-05-31T09:13:29.144Z">4885959589</value>
+  </datastream>
+  </interface>
+  <interface name="objectdatastreams.org" major_version="0" minor_version="1" active="true">
+  <datastream path="/objectendpoint1">
+  <object reception_timestamp="2019-06-11T13:24:03.200Z">
+  <item name="/y">2</item>
+  <item name="/x">45.0</item>
+  <item name="/d">78787985785</item>
+  </object>
+  <object reception_timestamp="2019-06-11T13:26:28.994Z">
+  <item name="/y">555</item>
+  <item name="/x">1.0</item>
+  <item name="/d">747989859</item>
+  </object>
+  <object reception_timestamp="2019-06-11T13:26:44.218Z">
+  <item name="/y">22</item>
+  <item name="/x">488.0</item>
+  <item name="/d">747847748</item>
+  </object>
+  </datastream>
+  </interface>
+  </interfaces>
+  </device>
+  </devices>
   </astarte>
   """
 
+  setup do
+    Xandra.Cluster.run(:astarte_data_access_xandra, fn conn ->
+      DatabaseTestHelper.seed_data(conn)
+    end)
+  end
```

Comment:
Usually `setup` blocks are below `setup_all`s, as `setup_all` runs before all tests in a case and `setup` runs before each test in a case. See https://hexdocs.pm/ex_unit/ExUnit.Callbacks.html#setup_all/2